includes/class-wc-gateway-ppec-checkout-handler.php

```diff
@@ -246,6 +246,9 @@ public function paypal_billing_details() {
 			wc_add_notice( $e->getMessage(), 'error' );
 			return;
 		}
+		if ( empty( $checkout_details->payer_details ) ) {
```

Comment:
A comment might help keep this function readable, just to be explicit about why we're doing this "extra" check first?